# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ruby/r12567.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

extern/C++ fix
http://svn.ruby-lang.org/cgi-bin/viewvc.cgi?view=rev&revision=12567

--- ruby-1.8.6-p36/intern.h_orig	2007-09-02 16:19:12.000000000 +0200
+++ ruby-1.8.6-p36/intern.h	2007-09-02 16:19:30.000000000 +0200
@@ -204,7 +204,6 @@
 void rb_thread_polling _((void));
 void rb_thread_sleep _((int));
 void rb_thread_sleep_forever _((void));
-enum rb_thread_status rb_thread_status _((VALUE));
 VALUE rb_thread_stop _((void));
 VALUE rb_thread_wakeup _((VALUE));
 VALUE rb_thread_wakeup_alive _((VALUE));
--- ruby-1.8.6-p36/ext/thread/thread.c_orig	2007-09-02 16:19:20.000000000 +0200
+++ ruby-1.8.6-p36/ext/thread/thread.c	2007-09-02 16:19:31.000000000 +0200
@@ -14,6 +14,8 @@
 #include <rubysig.h>
 #include <node.h>
 
+enum rb_thread_status rb_thread_status _((VALUE));
+
 static VALUE rb_cMutex;
 static VALUE rb_cConditionVariable;
 static VALUE rb_cQueue;
